### PR TITLE
added links

### PR DESCRIPTION
**Pull request by vermasavita**

### CODE REVIEW

**Review by Tejinder15 — CHANGES_REQUESTED** [review]
Fixes: Your Category section has an overflow on both axis.
Your new release section is not responsive try giving flex wrap.

**Review by GauravDe-git — APPROVED** [review]

**Comment by vermasavita** [comment]
@Tejinder15 Yea.. I am not focusing on responsiveness right now. But noted and will do it later.

File: account/login/login.html [hunk 1]
```diff
@@ -68,7 +68,7 @@ <h1>Login</h1>
 
     <div class="login-btns">
         <a href="#" class="login">Login</a>
-        <a href="/account/sign-up/signup.html" class="new-account">Create New Account <strong>></strong></a>
+        <a href="/account/sign up/signup.html" class="new-account">Create New Account <strong>></strong></a>
     </div>
```
Comment (shaik-saad) [inline account/login/login.html:72]: You've added a space in the name of folder, it's a good practice not to do so. Try avoiding using such naming conventions.

**Review by shaik-saad — APPROVED** [review]
Added a suggestion, other than that you're good to merge

File: account/login/login.html (hunk 1 above)
Comment (vermasavita) [reply account/login/login.html:72]: Noted

**Pull request merged**